### DISCUSSION TITLE
Request to add the possibility of adding proof photos from the mobile phone gallery

### DISCUSSION THREAD

**Discussion by maper64** [streetcomplete/StreetComplete#5344] [Ideas]
I found that I can only take a photo of the proof of the object's character, but I can't put the photo I already have on my mobile into the note. I can't see well and it's harder to read my mobile screen outside. Would it be possible to add photos to the note, which I will create later at home and add a photo? Thank you.

**Comment by matkoniecz** [comment]
See [FAQ](https://wiki.openstreetmap.org/wiki/StreetComplete/FAQ#Can_I_choose_photos_from_the_gallery_for_uploading_to_notes,_instead_of_using_camera?)

**Comment by maper64** [comment]
Díky, podívám se na to.

po 30. 10. 2023 v 19:07 odesílatel Mateusz Konieczny <
***@***.***> napsal:
